mindmeld/cli.py

```diff
@@ -445,6 +446,36 @@ def setup_blueprint(ctx, es_host, skip_kb, blueprint_name, app_path):
         ctx.exit(1)
 
 
+@module_cli.command('convert', context_settings=CONTEXT_SETTINGS)
+@click.pass_context
+@click.option('-d', '--df', is_flag=True, help="Convert a Dialogflow project")
+@click.option('-r', '--rs', is_flag=True, help="Convert a Rasa project")
+@click.argument('project_path', required=True, type=click.Path(exists=True))
+@click.argument('mindmeld_path', required=False)
+def convert(ctx, df, rs, project_path, mindmeld_path=None):
+    """Converts a Rasa or DialogueFlow project to a MindMeld project"""
+    try:
+        project_path = os.path.abspath(project_path)
+        mindmeld_path = os.path.abspath(mindmeld_path or "converted_app")
+        converter_cls = {'Rasa': RasaConverter, 'Dialogflow': DialogflowConverter}
+        if df:
+            framework = 'Dialogflow'
+        elif rs:
+            framework = 'Rasa'
+        else:
+            logger.warning("Please specify the project's platform Rasa/Dialogflow.")
```

Comment:
"Please specify the project's platform type (Rasa/Dialogflow)."

You can do this arg check outside the try block.